components/brave_wallet/browser/eth_data_builder.cc

```diff
@@ -79,7 +81,7 @@ std::optional<std::vector<uint8_t>> Forward(const FilAddress& fil_address) {
 
 namespace erc20 {
 
-bool Transfer(const std::string& to_address,
+bool Transfer(std::string_view to_address,
```

Comment:
`const std::string_view&`?

Reply:
`std::span` and `std::string_view` are view types, and supposed to be passed by value. Passing them by reference leads to worse codegen as it is cheaper to copy them over than to alias.